tests/gocase/go.mod

```diff
@@ -7,10 +7,13 @@ require (
 	github.com/stretchr/testify v1.8.0
 )
 
+require github.com/remyoudompheng/bigfft v0.0.0-20200410134404-eec4a21b6bb0 // indirect
+
 require (
 	github.com/cespare/xxhash/v2 v2.1.2 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/dgryski/go-rendezvous v0.0.0-20200823014737-9f7001d12a5f // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
+	modernc.org/mathutil v1.5.0
```

Comment:
I think you can put it in the first `require` clause, since it is not a indirect package.

Reply:
also fixed